lib/kubernetes-deploy/kubernetes_resource/horizontal_pod_autoscaler.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class HorizontalPodAutoscaler < KubernetesResource
+    PRUNABLE = true
```

Comment:
I do like the idea of defining this at the class level, but this doesn't do anything right now. Your test passes because, oddly enough, HPA was already on the whitelist.